charts/linkerd2/values.yaml

```diff
@@ -19,6 +19,9 @@ global:
 
   identityTrustDomain: *cluster_domain
 
+  # url of exisiting grafana instance. 
```

Comment:
Typo: existing
Or better yet, could you be more explicit and say something like "url for external Grafana instance"?